resources/js/tryitout.js

```diff
@@ -178,6 +178,21 @@ async function executeTryOut(endpointId, form) {
     const query = {};
     const queryParameters = form.querySelectorAll('input[data-component=query]');
     queryParameters.forEach(el => _.set(query, el.name, el.value));
```

Comment:
You know, I think this would have just been simpler:

```js
    queryParameters.forEach(el => {
        if (el.type !== 'radio' || (el.type === 'radio && el.checked)) {
            _.set(query, el.name, el.value));
        }
    });
```

I'll change it. Simpler is better. 😅